packages/block/examples/4844.ts

```diff
@@ -2,11 +2,14 @@ import { Common, Chain, Hardfork } from '@ethereumjs/common'
 import { Block } from '@ethereumjs/block'
 import { BlobEIP4844Transaction } from '@ethereumjs/tx'
 import { Address, initKZG } from '@ethereumjs/util'
-import * as kzg from 'c-kzg'
+import kzg from 'c-kzg'
 import { randomBytes } from 'crypto'
+import path from 'path'
+import { fileURLToPath } from 'url'
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url)).slice(0, -14)
```

Comment:
Why should this be sliced?

Reply:
`__dirname` as a global variable failed in the browser run, but worked in this way.  There's likely a more efficient way, but this slices off `/block/examples`  from the full file path, leaving you at `.../packages`